tbtadm/controller.cpp

```diff
@@ -80,19 +80,20 @@ const std::string indent     = "│   ";
 const size_t indentLength    = 4;
 const std::string indentLast = "    ";
 
-enum {
+enum
+{
     SYMBOL_PIPE,
     SYMBOL_L,
     SYMBOL_PLUS,
 };
 
 const std::string symbols[] = {
-    [SYMBOL_PIPE] = "│",
-    [SYMBOL_L] = "└─ ",
-    [SYMBOL_PLUS] = "├─ ",
+        [SYMBOL_PIPE] = "│",
+        [SYMBOL_L]    = "└─ ",
+        [SYMBOL_PLUS] = "├─ ",
 };
 
-const std::string green = "\x1b[0;32m";
+const std::string green  = "\x1b[0;32m";
```

Comment:
Do we need space before "=" ?

Reply:
It aligns consecutive assignments.
(See AlignConsecutiveAssignments section in https://clang.llvm.org/docs/ClangFormatStyleOptions.html)